workflow/platforms/hera.yaml

```diff
@@ -46,11 +46,11 @@ platform: !Platform
         scheduler_name: Slurm
         parallelism_name: HydraIMPI
         node_type: generic
-        physical_cores_per_node: 24
+        physical_cores_per_node: 20
```

Comment:
Not sure if it's a good idea, but we can go up to 40 cores/node as long we request all the memory of the node.

See here for the configuration  https://heradocs.rdhpcs.noaa.gov/wiki/index.php/Hera_system_configuration